style.css

```diff
@@ -383,3 +386,314 @@ input {
   margin-top: 6px;
   margin-left: 200px;
 }
+
+.desk-works {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
```

Comment:
- [OPTIONAL] Kindly make sure your mobile design still looks perfect as the provided Figma design. At this moment your mobile design doesn't look well, please fix that.

   - ### YOUR MOBILE DESIGN
   
    ![screencapture-127-0-0-1-5500-index-html-2023-03-30-17_25_40](https://user-images.githubusercontent.com/87197412/228886093-14aab810-9db1-42c2-b2f0-6b08747066e1.png)